test/tracing/test_set_direction.py

```diff
@@ -0,0 +1,76 @@
+#  Copyright 2021 Zeppelin Bend Pty Ltd
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from zepben.evolve import TestNetworkBuilder, Feeder, ConductingEquipment, set_direction, FeederDirection, NetworkService
+
+
+@pytest.mark.asyncio
+async def test_set_direction():
```

Comment:
Can you rename this to indicate it is testing that the upstream terminal of the feeder head is not set like you have for the dual feed version